nautobot/docs/configuration/optional-settings.md

```diff
@@ -1015,3 +992,33 @@ This setting function is used to perform the rack elevation truncation feature.
 The function must take only one argument: the device display name, as a string, attempting to be rendered on the rack elevation.
 
 The function must return only one argument: a string of the truncated device display name.
+
+## Environment-Variable-Only Settings
+
+!!! warning
+    The following settings are **only** configurable as environment variables, and not via `nautobot_config.py` or similar.
+
+---
+
+### GIT_SSL_NO_VERIFY
+
+Default: Unset
+
+If you are using a self-signed git repository, you will need to set the environment variable `GIT_SSL_NO_VERIFY="1"`
+in order for the repository to sync.
+
+!!! warning
+    This _must_ be specified as an environment variable. Setting it in `nautobot_config.py` will not have the desired effect.
+
+---
+
+### NAUTOBOT_ROOT
+
+Default: `~/.nautobot/`
+
+The filesystem path to use to store Nautobot files (Jobs, uploaded images, Git repositories, etc.).
+
+This setting is used internally in the core settings to provide default locations for [features that require file storage](../../configuration/#file-storage), and the [default location of the `nautobot_config.py`](../../configuration/#specifying-your-configuration).
+
+!!! warning
+    Do not override `NAUTOBOT_ROOT` in your `nautobot_config.py`. It will not work as expected. If you need to customize this setting, please always set the `NAUTOBOT_ROOT` environment variable.
```

Comment:
```suggestion
    Do not override `NAUTOBOT_ROOT` in your `nautobot_config.py`. It will not work as expected. If you need to customize this setting, please always set the `NAUTOBOT_ROOT` environment variable.

```